x509/cert_validations.go

```diff
@@ -184,10 +174,36 @@ func validateLeafKeyUsage(cert *x509.Certificate) error {
 		return err
 	}
 	if cert.KeyUsage&x509.KeyUsageDigitalSignature == 0 {
-		return fmt.Errorf("certificate with subject %q: key usage must have the bit positions for digital signature set", cert.Subject)
+		return fmt.Errorf("The certificate with subject %q is invalid. The key usage must have the bit positions for \"Digital Signature\"", cert.Subject)
```

Comment:
nit:
```suggestion
		return fmt.Errorf("The certificate with subject %q is invalid. The key usage must have the bit positions for \"Digital Signature\" set", cert.Subject)
```

Reply:
hmm, @shizhMSFT I guess we still need to display the current key usage of the invalid certificate?

Reply:
nvm, this is another error message.

Reply:
Updated.